### ISSUE TITLE
Map ReplayGain tags to common

### ISSUE THREAD

**Issue by Borewit** [Borewit/music-metadata#258]
**Feature Request**

 Map tags from [MP3Gain Express](http://projects.sappharad.com/mp3gain/) to `metadata.common`.

Related: 
* #192 _Normalize replay-gain values in common_ 
* #257 _APEv2 tags created by MP3Gain Express_
* PR: #260 _Map ReplayGain to common_

Reference documentation:
* https://wiki.hydrogenaud.io/index.php?title=ReplayGain#MP3Gain

**Comment by Borewit** [comment]
_Copied from https://github.com/Borewit/music-metadata/issues/257#issuecomment-531508413_

E.g. [aacgain](http://aacgain.altosdesign.com), the `mp3gain` pendant for AAC, creates the 4 gain tags as well, but not as APEv2 tags but as `iTunes` tags:

`tagTypes: [ 'iTunes' ]`

```
native: {
  iTunes: [
    ...
    { id: '----:com.apple.iTunes:replaygain_track_gain', value: '0.19' },
    { id: '----:com.apple.iTunes:replaygain_track_peak', value: '1.05' },
    { id: '----:com.apple.iTunes:replaygain_track_minmax', value: '67,174' },
    { id: '----:com.apple.iTunes:replaygain_undo', value: '-2,-2' }
  ]
}
```

**Comment by Borewit** [comment]
@jri

Are you sure it is?:
```
{ id: '----:com.apple.iTunes:replaygain_track_minmax', value: '67,174' },
{ id: '----:com.apple.iTunes:replaygain_undo', value: '-2,-2' }
```
and not
```
{ id: '----:com.apple.iTunes:mp3gain_track_minmax', value: '67,174' },
{ id: '----:com.apple.iTunes:mp3gain_undo', value: '-2,-2' }
```

Do you have test file by any chance?

**Comment by jri** [comment]
De facto `aacgain` creates `replaygain_track_minmax` and `replaygain_undo` tags.
See attached file: [06 Gbeti Madjiro.m4a.zip](https://github.com/Borewit/music-metadata/files/3613833/06.Gbeti.Madjiro.m4a.zip)

Yes, that's indeed a different wording than in the APEv2 tags.
I guess there are just de facto "standards" for these gain related tag names.
While "ReplayGain" is the name of the algorithm "mp3gain" is the name of an implementation.

**Comment by Borewit** [comment]
Released in [v4.6.0](https://github.com/Borewit/music-metadata/releases/tag/v4.6.0)
